### ISSUE TITLE
Module '"../node_modules/@types/testing-library__react"' has no exported member 'screen'

### ISSUE THREAD

**Issue by Nnadozie** [testing-library/react-testing-library#802]
- `@testing-library/react` version: 9.5.0
- Testing Framework and version:     "jest": "^25.1.0", 
  
- DOM Environment: jest-environment-jsdom@^25.5.0, version 25.5.0
  

### Relevant code or config:

```js
import { render, screen } from "@testing-library/react";
```

### What happened: 
worked fine, except I noticed a typescript warning saying the screen type is not exported. 
![image](https://user-images.githubusercontent.com/15310842/96319388-6e85a580-1007-11eb-879a-744a704d049c.png)

### Reproduction: 
With Typescript static checking enabled, import screen and try to check it's type definition using intellisense.

### Problem description: 
Not a major issue, the screen functionality works anyway, but I just dislike having that warning.

### Suggested solution: 
I'd like to make a pr with some guidance updating types/index.d.ts to include an export of the screen definition.

**Comment by eps1lon** [comment]
What version of `@testing-library/dom` have you installed? If you're using `yarn` you can run `yarn why @testing-library/dom` to find out.

> @testing-library/react version: 9.5.0

That version is already 2 major versions behind. The easiest solution would be to bump it. That way you get the latest version of `@testing-library/dom`

**Comment by Nnadozie** [comment]
Turns out I had ``@testing-library/dom@6.16.0``.

I had actually previously tried an upgrade (didn't mention, sorry) using ``yarn upgrade`` to upgrade the dependency versions which my Nextjs with Jest example came with, and so I thought 9.5.0 was the latest 🤦‍♂️ (should have checked, sorry).

This time I upgraded to latest versions using ``yarn upgrade --latest``, instead of just ``yarn upgrade``.

My current versions are now ``@testing-library/dom@7.26.0`` and ``@testing-library/react@11.1.0`` and the issue is resolved in these versions.

Thank you ❤️
